### DISCUSSION TITLE
Can I change the directory for #[ts(export)] -exported stuff?

### DISCUSSION THREAD

**Discussion by Niedzwiedzw** [Aleph-Alpha/ts-rs#66] [Q&A] [answered]
it's `./bindings` by default, I'd rather have it save the bindings directly in my front-end's directory

**Comment by NyxCode** [comment, marked as answer]
Yes. You can use `#[ts(export_to = "../frontend/src/MyType.ts")]` for this.  
Currently, you'll have to do this for every type. In the future, we want to support more fine-grained configuration through a config file. This is sadly blocked by [this](https://github.com/rust-lang/rust/issues/73921)

**Comment by NyxCode** [comment]
This is documented here, btw: https://docs.rs/ts-rs/latest/ts_rs/trait.TS.html#container-attributes

**Reply by Niedzwiedzw** [reply]
hmm too bad I can't specify just the directory, I'm gonna have to repeat the struct name everywhere...

**Reply by NyxCode** [reply]
This is now supported - you can now also just specify a directory.
